Vertexvis/vertex-web-tools
# [SDK-637] Support for connecting to FSS

## What

Adds support for connecting to FSS by adding a `streamingClient` option that can be passed with the network configuration. Setting this to `platform` will open a connection to FSS instead of ISS.

As part of setting up the ability to load a scene, support for a new `urn:vertexvis:platform:scene:id` URN was added.

Note: currently, some of the existing behavior around manipulating scenes will not be functional until we complete https://jira.vertexvis.net/browse/SDK-637.

## Testing

- Configure the SDK to point at `platdev` 
```
viewer.config = {
  network: {
    apiHost: 'https://platform.platdev.vertexvis.io',
    renderingHost: 'wss://stream.platdev.vertexvis.io',
    streamingClient: 'platform'
  },
};
```
- Provide a scene id through `viewer.load('urn:vertexvis:platform:scene:id')`
- Verify an image is rendered

@Vertexvis/sdk 
PTAL
